### PR TITLE
fix: disable scroll on react calendar on profile stats page

### LINKED ISSUE #4191: Disable scroll on react calendar on profile stats page
Image

### PR DESCRIPTION
**Pull request by neo773**
Disables scroll on react calendar on profile stats page
This also auto scrolls the calendar on x axis so only recent months are shown as disabling scroll would hide them

https://github.com/heyxyz/hey/assets/62795688/0d2fcd36-77a8-475c-bb87-4b25eece3473

/fixes #4191
/claim #4191

### CODE REVIEW

**Comment by bigint** [comment]
@neo773 I dont see december here

Hey Prod

Screenshot 2023-12-12 at 4 26 39 PM

Preview

Screenshot 2023-12-12 at 4 26 52 PM

**Comment by neo773** [comment]
@bigint 
Like I said we're scrolling to right side of the element, wouldn't make sense to show older months and hide the newer ones would it?

**Comment by rishi-raj-jain** [comment]
@bigint @neo773 

How about using CSS on the parent div to hide the scrollbar but the functionality of scroll be there?

**Comment by bigint** [comment]
Ummm not sure, the best would be dynamic sizing of the grid

**Comment by neo773** [comment]
@bigint 
Found an easier pure CSS fix, the grid is now sized dynamically

File: apps/web/src/styles.css
Comment (bigint) [inline apps/web/src/styles.css:1]: seems like prettier not got applied in this file!

File: apps/web/src/components/Profile/Achievements/Streaks.tsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import type { Profile } from '@hey/lens';
-import type { FC } from 'react';
 import type { Activity } from 'react-activity-calendar';
 
 import { HEY_API_URL } from '@hey/data/constants';
 import getProfile from '@hey/lib/getProfile';
 import { Card } from '@hey/ui';
 import { useQuery } from '@tanstack/react-query';
 import axios from 'axios';
+import { type FC } from 'react';
```
Comment (bigint) [inline apps/web/src/components/Profile/Achievements/Streaks.tsx:9]: types should be always declared above!

**Review by bigint — CHANGES_REQUESTED** [review]

**Comment by bigint** [comment]
Screenshot 2023-12-14 at 10 39 55 AM

im still seeing this, github handles it nicely!

**Comment by bigint** [comment]
Fixed in https://github.com/heyxyz/hey/pull/4228

**Pull request closed without merging**